state/mongodb/mongodb.go

```diff
@@ -416,6 +417,10 @@ func (m *MongoDB) Query(ctx context.Context, req *state.QueryRequest) (*state.Qu
 }
 
 func getMongoURI(metadata *mongoDBMetadata) string {
```

Comment:
Can you rename to `getMongoDBConnString`?

Reply:
Use the suggestion feature so I can apply it in the future?